### PR TITLE
[release/1.8] Upgrade lite subgraph api.

### PR DESCRIPTION
**Pull request by jiweibo**
### PR types

Others
### PR changes

Others
### Describe

lite xpu接口升级，子图xpu接口对应修改

### CODE REVIEW

**Review by Shixiaowei02 — DISMISSED** [review]

File: paddle/fluid/inference/api/analysis_predictor.cc [hunk 1]
```diff
@@ -462,6 +462,12 @@ void AnalysisPredictor::PrepareArgument() {
     argument_.SetLiteZeroCopy(config_.lite_zero_copy_);
     argument_.SetUseXpu(config_.use_xpu_);
     argument_.SetXpuL3WorkspaceSize(config_.xpu_l3_workspace_size_);
+    argument_.SetXpuLocked(config_.xpu_locked_);
+    argument_.SetXpuAutotune(config_.xpu_autotune_);
+    argument_.SetXpuAutotune(config_.xpu_autotune_);
```
Comment (zhupengyang) [inline paddle/fluid/inference/api/analysis_predictor.cc]: 重复了

**Review by hong19860320 — DISMISSED** [review]
LGTM

File: paddle/fluid/inference/api/analysis_predictor.cc (hunk 1 above)
Comment (jiweibo) [reply paddle/fluid/inference/api/analysis_predictor.cc]: done

**Review by Shixiaowei02 — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**